client/my-sites/plans/current-plan/index.jsx

```diff
@@ -106,6 +107,9 @@ class CurrentPlan extends Component {
 		const currentPlanSlug = selectedSite.plan.product_slug,
 			isLoading = this.isLoading();
 
+		const currentPlanThankYouCard =
+			currentPlanSlug === 'jetpack_free' ? <FreePlanThankYouCard /> : <PaidPlanThankYouCard />;
```

Comment:
I'd discourage using plan slugs in application code like this, it tends to be fragile.

We have selectors like this:

https://github.com/Automattic/wp-calypso/blob/844807722fc7a5cb9f9b3bd03ecc48450db3e67c/client/state/selectors/is-site-on-free-plan.js#L20

There are also utilities like those used here:

https://github.com/Automattic/wp-calypso/blob/844807722fc7a5cb9f9b3bd03ecc48450db3e67c/client/my-sites/checkout/checkout-thank-you/jetpack-thank-you-card.jsx#L262-L263

Reply:
Nice one; changed to use selector in https://github.com/Automattic/wp-calypso/pull/32827/commits/b1e0362935782212d413ff314258bef321a25e05